llvm/lib/SYCLLowerIR/LowerKernelProps.cpp

```diff
@@ -47,9 +47,7 @@ void processSetKernelPropertiesCall(CallInst &CI) {
     // TODO: Keep track of traversed functions to avoid repeating traversals
     // over same function.
     llvm::esimd::traverseCallgraphUp(F, [](Function *GraphNode) {
```

Comment:
This function is in esimd namespace. is this ok?

Reply:
yeah good catch, let me move that function too